client/src/app/pages/controls/tags/components/tag-table.tsx

```diff
@@ -41,17 +34,9 @@ export const TagTable: React.FC<TabTableProps> = ({
             <Tr key={tag.name}>
               <Td>{tag.name}</Td>
               <Td isActionCell>
```

Comment:
We shouldn't wrap `ControlTableActionButtons` with  `<Td isActionCell>` as it already contains table cells inside.